app/templates/components/recruiter-conversations/conversation.hbs

```diff
@@ -1,25 +1,26 @@
 {{#paper-card as |card|}}
   {{#card.title as |title|}}
     {{#title.text as |text|}}
-      {{#text.headline}}
-        You invited {{conversation.engineer.name}} to lunch!
-      {{/text.headline}}
+    <div class="face-container">
+      {{paper-icon "person" size=88}}
+    </div>
+    {{#text.subhead}}
+     <h2>You invited {{conversation.engineer.name}} to lunch!</h2>
+    {{/text.subhead}}
     {{/title.text}}
     {{#title.media size="lg" }}
       {{recruiter-conversations/conversation/status conversation=conversation}}
     {{/title.media}}
   {{/card.title}}
 
   <div class="md-actions" layout-align="center">
-    <a target="_blank" href="{{conversation.engineer.website~}}">
-      {{#paper-button}}View their website{{/paper-button}}
-    </a>
+    {{!-- inconsistent syntax due to a problem with using a paper-button with an href attribute
+    filed an issue on the ember-paper github --}}
```

Comment:
The new button syntax in 1.0 works in other places but not here.
https://github.com/miguelcobain/ember-paper/issues/748